bindings/jupyter/js/webpack.config.js

```diff
@@ -24,7 +24,7 @@ module.exports = [
     output: {
       path: resolve('../mlvis/static'),
       filename: 'index.js',
-      libraryTarget: 'umd',
+      libraryTarget: 'amd',
```

Comment:
any concerns on non-amd targets?

Reply:
Since the the JS bundle is only intended to be used inside Jupyter, it is better to use amd to get a smaller bundle size.